docker-compose.yml

```diff
@@ -39,5 +39,7 @@ services:
     # The following turns the DB into less durable, but gains significant performance improvements for the tests run (x3
     # improvement on my personal machine). We should consider moving this into a dedicated Docker Compose configuration for
     # tests.
+    ports:
+      - "15432:5432"
```

Comment:
☝️

Reply:
Yeah, so question: I often do that to inspect the database during development using a macOS app ([Postico](https://eggerapps.at/postico/)). Do you mind having that exposed for development?

Reply:
You could use Redash to inspect it ;-) But sure, I guess there is no harm in leaving this.

Reply:
Heh, I do that, too, in some cases. But in others it's easier to just use other software, plus it allows editing as well (e.g. reverting archiving queries and dashboards).